dayPlanner.html

```diff
@@ -17,72 +17,73 @@
               </tr>
             </thead>
             <tbody>
-              <tr onclick="{clickableTR1()}">
+              <tr id="tr1" onclick="{funcGroup.func1()}">

```

Comment:
- burada `clickableTR1` fonksiyonları tek tek tüm `tr` targetlerını işliyordu
- bunu bir `for` loopu içine alıp tüm fonksiyonları 2-3 kod bloğuna indirgiyoruz